BUILD

```diff
@@ -28,7 +29,7 @@ NODE_MODULES_EXCLUDE = [
 filegroup(
     name = "node_modules",
```

Comment:
Is this glob still needed? I thought it wasn't with the fine grained deps.

Reply:
good catch, it should be removed